Jucer2Reprojucer/main.cpp

```diff
@@ -145,6 +146,45 @@ int main(int argc, char* argv[])
     return 1;
   }
 
+  const std::string juceCommitSha1 = args.size() == 4 ? args.at(3) : std::string{};
+  const auto commitSha1 = [&juceCommitSha1]() -> unsigned long
+  {
+    if (juceCommitSha1.empty())
+      return kDefaultCommitSha1;
+
+    if (juceCommitSha1.length() < 7)
+      return 0u;
+
+    static_assert(sizeof(unsigned long) >= 7 * 4 / 8, "");
```

Comment:
Missing error message (this sizeof is cryptic, to what command is it associated with ?)